Doc/howto/instrumentation.rst

```diff
@@ -332,6 +332,15 @@ Available static markers
    .. versionadded:: 3.7
 
 
+.. c:function:: audit(str event, void *tuple)
+
+   Fires when :func:`sys.audit` or :c:func:`PySys_Audit` is called.
+   ``arg0`` is the event name as C string, ``arg1`` is a :c:type:`PyObject`
```

Comment:
Does C string mean "char*"? What's the encoding of the event name? UTF-8 I guess? Or ASCII?

Reply:
This is a USDT probe point, not actually a C function. We just use ``c:function`` to encode USDT. It's a NULL terminated string array that contains a user space string.